utils/translationKeys.txt

```diff
@@ -0,0 +1,106 @@
+Padding                                                                                                      // /DefaultEditor.js:216
+Points                                                                                                       // /components/containers/TraceMarkerSection.js:22
+Position                                                                                                     // /DefaultEditor.js:272
+Positioning                                                                                                  // /DefaultEditor.js:384
+Range                                                                                                        // /DefaultEditor.js:309
+Relative To                                                                                                  // /DefaultEditor.js:271
+Reversed                                                                                                     // /DefaultEditor.js:440
+Rich Text                                                                                                    // /components/widgets/text_editors/MultiFormatTextEditor.js:24
+Rich text is incompatible with LaTeX.                                                                        // /components/widgets/text_editors/MultiFormatTextEditor.js:120
+Right                                                                                                        // /DefaultEditor.js:215
+Scale                                                                                                        // /DefaultEditor.js:241
+Selection                                                                                                    // /DefaultEditor.js:311
+Shape                                                                                                        // /DefaultEditor.js:160
+Show                                                                                                         // /DefaultEditor.js:233
+Size                                                                                                         // /DefaultEditor.js:131
+Size and Spacing                                                                                             // /DefaultEditor.js:137
+Symbol                                                                                                       // /DefaultEditor.js:132
+Text                                                                                                         // /DefaultEditor.js:364
+Text Attributes                                                                                              // /DefaultEditor.js:82
+The anchor point determines which side of the annotation's positioning coordinates refer to.                 // /DefaultEditor.js:256
+The positioning inputs are relative to the anchor points on the text box                                     // /DefaultEditor.js:389
```

Comment:
`.`

Reply:
This is a nice way to correct all the text in a program.

Reply:
yup, agreed, making corrections now :)